www/src/components/email-capture-form.js

```diff
@@ -1,9 +1,182 @@
 import React from "react"
 import { rhythm, options } from "../utils/typography"
-import { colors } from "../utils/presets"
-import HubspotForm from "./hubspot-form"
+import presets, { colors } from "../utils/presets"
+import hex2rgba from "hex2rgba"
+import { formInput } from "../utils/form-styles"
+import { buttonStyles } from "../utils/styles"
+
+const Label = props => (
+  <label>
```

Comment:
I forgot to use `for` prop here

Reply:
htmlFor, I forget it almost every time 🙃

Reply:
heh, I've initially used `for` and of course got complaints about syntax problem so as any dev I just deleted that and forgot about it :D